test_site/index.html

```diff
@@ -6,15 +6,26 @@
 
 <title>DAP test site</title>
 <script>
-	(function() {
+  (function() {
 		const queryParams = new URLSearchParams(window.location.search);
-		const dapScriptTag = document.createElement("script");
-		dapScriptTag.id = "_fed_an_ua_tag";
-		dapScriptTag.async = true;
-		dapScriptTag.src = `Universal-Federated-Analytics-Min.js?dapdev=true&${queryParams.toString()}`;
-		document.head.appendChild(dapScriptTag);
-	})();
+    const dapScriptTag = document.createElement("script");
+		let dapCodeLocation;
+    dapScriptTag.id = "_fed_an_ua_tag";
+    dapScriptTag.async = true;
+
```

Comment:
There aren't any automated tests hitting it yet but I guess this would need to be replicated on the `youtube.html` page too.